trello_functions.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from item import Item
+
+auth_params = {'key' : 'b7ff82f3970592f40825cfb7c771881c', 'token' : 'b3918107d6c9333fcc87a9b0a362b1801db711b71559f152a724b63d5b070b62'}
```

Comment:
Keys should not be committed into source control. This is task 2 on the exercise.

You will probably want to add these to the .env file, pulled in via os.environ.get() and documented in README (so somebody else could wire their own trello board up to this app).